students/BrianNisonger/session9/mailroom.py

```diff
@@ -0,0 +1,197 @@
+import statistics
+
+
+class Donor():
+    """ Class used to contain donor name,donation, and methods involved with a single donor """
+
+    def __init__(self, name, donation=""):
+        """ Takes a name, and optionally a donation list"""
+        self.name = name
+        self.donation = list(donation)
+
+    def add_donation(self, *values):
+        """Takes either a single donation, and appends it to the list, or appends a set of donations"""
+        for value in values:
+            self.donation.append(value)
```

Comment:
This could be just `self.donation.extend(values)` 

Have you tested with single numbers and multiple numbers?